internal/cmd/load-balancer/target-pool/add-target/add_target.go

```diff
@@ -0,0 +1,147 @@
+package addtarget
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/load-balancer/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/load-balancer/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/loadbalancer"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	loadBalancerNameArg = "LOAD_BALANCER_NAME"
+
+	targetPoolNameFlag = "target-pool-name"
+	targetNameFlag     = "target-name"
+	ipFlag             = "ip"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+
+	LoadBalancerName string
+	TargetPoolName   string
+	TargetName       string
+	Ip               string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("add-target %s", loadBalancerNameArg),
+		Short: "Adds a target to a target pool",
+		Long:  "Adds a target to a target pool.",
```

Comment:
Added that the IP should be a valid IPv4 or IPv6 to the flag's description. The point that the target pool name should be unique I think it would belong to load balancer create/update, as this just allows to update a target in a target pool. But even then, I'm not sure if that should be something we add to the CLI command description, as we already give a link to the docs of the expected payload